src/Translator.php

```diff
@@ -150,82 +152,96 @@ protected function loadLanguageStrings($locale, $platform = null, $force = false
             $platform = $this->getDefaultPlatform();
         }
 
-        if (in_array($locale, $this->languages)) {
-            if ($force or !isset($this->strings[$locale])) {
-                $client = $this->getClient();
+        if (is_null($check)) {
+            $check = !is_null($this->getUpdateAfter());
```

Comment:
Defines wheter or not to check for string updates.

If `update_after` is `null` then it will not automatic update strings.